apis/v1alpha2/validation/httproute_test.go

```diff
@@ -19,43 +19,41 @@ package validation
 import (
 	"testing"
 
+	"k8s.io/apimachinery/pkg/util/validation/field"
 	utilpointer "k8s.io/utils/pointer"
 
 	gatewayv1a2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
+	pkgutils "sigs.k8s.io/gateway-api/pkg/util"
```

Comment:
Nit: Maybe just import this as "gwutils" or something to make it clearer that these utils are specific to this project.

Reply:
"pkgutils" is accurate - the utils under pkg/utils directory. leave as it is.